### PR TITLE
Gauntlet Multisig - Reduce create proposal TX size

### PR DESCRIPTION
**Pull request by RodrigoAD**

### CODE REVIEW

File: gauntlet/packages/gauntlet-serum-multisig/networks/.env.local [hunk 1]
```diff
@@ -1 +1,10 @@
-NODE_URL=http://127.0.0.1:8899
\ No newline at end of file
+NODE_URL=http://127.0.0.1:8899
+
+PROGRAM_ID_MULTISIG=3X4UvrX9gTxSmNmXww53W5dM5fT1Nby246hKV8Mx9LuD
```
Comment (ebarakos) [inline gauntlet/packages/gauntlet-serum-multisig/networks/.env.local:3]: Remove these local vars?

File: gauntlet/packages/gauntlet-solana-contracts/artifacts/schemas/serum_multisig.json [hunk 2]
```diff
@@ -0,0 +1,322 @@
+{
```
Comment (ebarakos) [inline gauntlet/packages/gauntlet-solana-contracts/artifacts/schemas/serum_multisig.json:1]: Hard-add it here or let it be added along other programs and .so's during Gauntlet setup?

File: gauntlet/packages/gauntlet-serum-multisig/src/commands/multisig.ts [hunk 3]
```diff
@@ -95,44 +102,46 @@ export const wrapCommand = (command) => {
...
-            {
-              tx: this.wrapResponse(tx, proposal.toString()),
-              contract: proposal.toString(),
-            },
-          ],
-        } as Result<TransactionResponse>
-      } catch (e) {
-        // known errors, defined in multisig contract. see serum_multisig.json
-        if (e.code >= 300 && e.code < 400) {
-          logger.error(e.msg)
```
Comment (ebarakos) [inline gauntlet/packages/gauntlet-serum-multisig/src/commands/multisig.ts:112]: Why removing this logging for known erros?

File: gauntlet/packages/gauntlet-serum-multisig/src/commands/multisig.ts [hunk 4]
```diff
@@ -95,44 +102,46 @@ export const wrapCommand = (command) => {
...
+          },
+        ],
       }
     }
 
-    createProposal: ProposalAction = async (proposal: Keypair, context): Promise<string> => {
+    createProposalAcount = async (): Promise<PublicKey> => {
+      logger.log('Creating proposal account')
+      const proposal = Keypair.generate()
+      const txSize = 1300 // Space enough
```
Comment (ebarakos) [inline gauntlet/packages/gauntlet-serum-multisig/src/commands/multisig.ts:123]: Nice catch

**Review by ebarakos — CHANGES_REQUESTED** [review]

File: gauntlet/packages/gauntlet-serum-multisig/src/commands/multisig.ts (hunk 3 above)
Comment (RodrigoAD) [reply gauntlet/packages/gauntlet-serum-multisig/src/commands/multisig.ts:112]: This wasn't doing anything. The program already logs this errors automatically

File: gauntlet/packages/gauntlet-solana-contracts/artifacts/schemas/serum_multisig.json (hunk 2 above)
Comment (RodrigoAD) [reply gauntlet/packages/gauntlet-solana-contracts/artifacts/schemas/serum_multisig.json:1]: Let's have it here, in the future will be pulled from GH. But for development is useful

File: gauntlet/packages/gauntlet-serum-multisig/networks/.env.local (hunk 1 above)
Comment (RodrigoAD) [reply gauntlet/packages/gauntlet-serum-multisig/networks/.env.local:3]: Those are constant always on local environment

Comment (ebarakos) [reply gauntlet/packages/gauntlet-serum-multisig/networks/.env.local:3]: PROGRAM_IDs yes, but MULTISIG_ADDRESS is created anew each time, no?

Comment (RodrigoAD) [reply gauntlet/packages/gauntlet-serum-multisig/networks/.env.local:3]: That's right. Removing those

**Review by ebarakos — APPROVED** [review]

**Pull request merged**